src/main/java/org/opensearch/jobscheduler/transport/action/TransportGetScheduledInfoAction.java

```diff
@@ -138,12 +138,12 @@ protected GetScheduledInfoNodeResponse nodeOperation(GetScheduledInfoNodeRequest
                                 // Add execution information
 
                                 if (jobInfo.getActualPreviousExecutionTime() != null) {
-                                    jobDetails.put("last_execution_time", jobInfo.getActualPreviousExecutionTime());
+                                    jobDetails.put("last_execution_time", jobInfo.getActualPreviousExecutionTime().toString());
```

Comment:
I think we should use a DateTime formatter similar to core: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java#L1073